src/main/java/com/aws/greengrass/status/FleetStatusService.java

```diff
@@ -545,6 +567,11 @@ private Topic getPeriodicUpdateTimeTopic() {
         return config.lookup(FLEET_STATUS_LAST_PERIODIC_UPDATE_TIME_TOPIC).dflt(Instant.now().toEpochMilli());
     }
 
+    private Topic getReconnectUpdateTimeTopic() {
+        // default to zero so that first Reconnect update would go thru
+        return config.lookup(FLEET_STATUS_LAST_RECONNECT_UPDATE_TIME_TOPIC).dflt(0);
+    }
```

Comment:
Why do you need to maintain this in config?

Reply:
Updated to only store in memory